Add copybara@0.0.0-20250728

zifeitong opened the pull request.

bazel-io commented:
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (copybara) have been updated in this PR.
      Please review the changes. You can view a diff against the previous version in the "Generate module diff" check.

kotlaja commented:
@zifeitong, previous version of this module is "0.0.0-20240326-6485af6", while this one is added as "0.0.0-20250728" and it is missing the last part - Can you please add this?

This seems to be the reason of a false positive error in the CI: `BcrValidationResult.FAILED: copybara's metadata.json file is not sorted by version.`, even thou I sorted it.

Also, would you maybe like to be a maintainer for this module? :)
Modules with no maintainers always require BCR maintainer review, which adds to response times and toil.

zifeitong commented:
> @zifeitong, previous version of this module is "0.0.0-20240326-6485af6", while this one is added as "0.0.0-20250728" and it is missing the last part - Can you please add this?

I added the commit hash. I removed it since upstream started to provide releases in vYYYYMMDD format.

> This seems to be the reason of a false positive error in the CI: `BcrValidationResult.FAILED: copybara's metadata.json file is not sorted by version.`, even thou I sorted it.
> 
> Also, would you maybe like to be a maintainer for this module? :) Modules with no maintainers always require BCR maintainer review, which adds to response times and toil.

Happy to do it. Updated metadata.json.

zifeitong commented:
@bazel-io skip_check incompatible_flags
